patches/0004-Add-OpenSSL-crypto-backend.patch

```diff
@@ -281,6 +282,10 @@ index 00000000000000..d929b3e516bdb0
 +			}
 +		}
 +	}
++	if openssl.FIPS() {
++		fipstls.Force()
++		sig.FIPSOnly()
```

Comment:
Hmm, interesting. Given that we can't really make `sig.FIPSOnly` work correctly, I would rather not add it. If someone is using an scanner looking for `sig.FIPSOnly` as a matter of compliance, I prefer to have a false negative (aka binary reported as not compliant when it is compliant) that can be fixed with an exception rather than a false positive (aka binary reported as compliant when it is not compliant).